### ISSUE TITLE
Select-folder property

### ISSUE THREAD

**Issue by jebeaudoin** [tgrosinger/tq-obsidian#12]
**Is your feature request related to a problem? Please describe.**
Hi Tony. Following development here with excitement. My current challenge implementing tq is that I organize and differentiate projects by folder, not tag. Thus, I don't know how I might be able to build a view of tasks that are related to specific projects.

**Describe the solution you'd like**
Are you considering adding a "select-folder" property in the future? 

**Describe alternatives you've considered**
Without a folder-based property, I would have to create a tag for each project folder just to achieve this. Not impossible, but unwieldy.

**Additional context**
I really, really prefer the file-based task approach (as opposed to a block-based task) and hope that you are able to continue with your work! Thanks!

**Comment by jebeaudoin** [comment]
Ahhh. Just recalled that tq uses a 'tasks' folder so my implementation approach fails. Well, I'd modify my request to support another user's request to move tasks into any folder within the vault and then use a "select-folder" property :)

**Comment by tgrosinger** [comment]
I worry that putting these tasks in other folders would cause a bit of a mess. Right now they are all named `task-XXXX` where the last four characters are random. The files are meant to be pretty invisible and not interacted with directly. They just need to be in the vault somewhere.

**Comment by jebeaudoin** [comment]
Fair enough. I wonder if you might consider a "select-link" property (the dataview plugin has both an in-link and out-link property) as an alternative. That would provide the specificity strength of a folder location without moving tasks into a folder.

Not to go too far astray, but on the topics of tags... As of v0.1.1. if I want to add multiple tags in the dialogue, I can, but only the first triggers the autocomplete function. If I do change my workflow to create tags for each project, autocomplete for all tags would be very helpful.

**Comment by tgrosinger** [comment]
Thanks for letting me know about the tag autocomplete. I will file another issue for that.

I'll consider the select-link idea. Thanks 🙂

**Comment by tgrosinger** [comment]
By `select-link` do you mean select tasks which contain a link to a specific note?

**Comment by jebeaudoin** [comment]
Yes. The link could be inline (same block as the task) or YAML I suppose
